### DISCUSSION TITLE
[Feature request] Add File Explorer application

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#372] [Feature Requests]
What I mean by this is that other than the start menu shortcuts, and the "My PC" and "Public" icons on the desktop, there are no other "File Explorer" application shortcuts like Windows has.

**Comment by DustinBrett** [comment]
Thanks for the suggestion. The File Explorer can be launched via the Run dialog or making a shortcut, but it's default behavior is to open "/" which is "My PC" and is a shortcut on the desktop already. Because of the My PC icon on the desktop I am not sure if there would be much value in the File Explorer shortcut too.
